packages/workspace/src/tasks-runner/reporter.ts

```diff
@@ -6,6 +6,9 @@ export interface ReporterArgs {
   onlyFailed?: boolean;
 }
 
+/**
+ * @deprecated the reporter API is no longer supported
```

Comment:
Shall we add an explicit note about being removed in `v14` so that it is easier to grep for in the future when preparing for that release?